package.json

```diff
@@ -82,7 +86,7 @@
     "edition": "octane"
   },
   "volta": {
-    "node": "16.16.0",
+    "node": "14.20.0",
```

Comment:
It is not common to include package.json file in commits unless you intend to change package. Please reconfirm.

Reply:
Hi rajakvk, I have  added some package's that why it is showing  changes in package.json file.

Reply:
Looks like there are changes in  webpack, bootstrap as well as node version. Hope it is intentional.

Reply:
Yes the changes are intentional.
1. The change in webpack and bootstrap are because they are needed to run this https://www.ember-bootstrap.com/  package.
2. The change in the node because when I tried to run this project locally it was shwoing an error that it will work only on node 14 so I switched to node 14 but after your last comment I tried to re-run it with node 16 and it works so I changed the node back to 16.

Reply:
Thanks for confirming.